### DISCUSSION TITLE
"Kepler : mean", What is meant by "mean" ?

### DISCUSSION THREAD

**Discussion by dingxu6207** [phoebe-project/phoebe2#629] [Questions]
I am sorry, I want to know  the information about "Kepler : mean.". What is meant by "mean" ?
Does it refer just to mean wavelength? Or is the flux integrated over the entire passband and appropriately
weighted? After all, the passband is rather wide.

**Comment by aprsa** [comment]
Each Kepler channel (a set of 2 CCDs) has its own response function. 
Instead of keeping track of per-channel transmission, all, transmissions 
are averaged into the mean response function, thus named Kepler:mean.

**Reply by dingxu6207** [reply]
Many thanks!
